assets/out

```diff
@@ -89,6 +89,14 @@ if [ -n "$load" ]; then
   docker load -i "${load}/image"
   docker tag $(cat "${load}/image-id") "${repository}:${tag_name}"
 elif [ -n "$build" ]; then
+  ECR_REGISTRY_PATTERN='[0-9]\+\.dkr.ecr.\+amazonaws\.com'
```

Comment:
This should probably more closely match:

https://github.com/awslabs/amazon-ecr-credential-helper/blob/407daf548f87666c4f6cbefe79bf521f9a7f626f/ecr-login/ecr.go#L27

In particular it looks like the subdomain can be non-numeric.